Automattic/wp-calypso
# Eligibility Checks: add launch button for unlaunched sites

Fixes https://github.com/Automattic/wp-calypso/issues/39408

Instead of linking to a support page, let's get out of the way and let customers perform whatever action is stopping AT. To start, this PR will focus on launching a site.

Before:
<img width="847" alt="74333058-11a66100-4d8f-11ea-9633-9803ff4d2780" src="https://user-images.githubusercontent.com/1270189/74888949-61a79980-5334-11ea-804b-5b314b87b9cc.png">

After: (Resolve-able holds are buying a plan, then launching a site).
<img width="1098" alt="Screen Shot 2020-02-20 at 11 11 30 AM" src="https://user-images.githubusercontent.com/1270189/74969713-e7c5ed80-53d1-11ea-9f93-7a523f86ecfe.png">

![Feb-20-2020 11-09-52](https://user-images.githubusercontent.com/1270189/74969717-e98fb100-53d1-11ea-9f1a-271aa355dfd6.gif)

### Testing Instructions
- Create a new site, it will be unlaunched by default
- Visit /hosting or one of the other AT points (plugins, or themes)
- On a free site, try to activate hosting, upload a plugin, or install a theme
- Verify that clicking on launch site, launches a site, and allows customers to proceed through next AT steps
- Verify no regressions from the Launch Site button in customer home (/home) 
- Verify that a **launched** site that is set to private still sees the following:
<img width="847" alt="74333058-11a66100-4d8f-11ea-9633-9803ff4d2780" src="https://user-images.githubusercontent.com/1270189/74891463-f661c580-533b-11ea-80e5-60604a5010bb.png">
